curriculum/challenges/english/08-coding-interview-prep/rosetta-code/sha-256.english.md

```diff
@@ -0,0 +1,411 @@
+---
+id: 5eb3e4c3a894c333d2811a3f
+title: SHA-256
+challengeType: 5
+forumTopicId: 385328
+---
+
+## Description
+
+<section id='description'>
+<b><a href="https://en.wikipedia.org/wiki/SHA-256" target="_blank">SHA-256</a></b> is the recommended stronger alternative to <a href="http://rosettacode.org/wiki/SHA-1" target="_blank">SHA-1</a>.
+</section>
+
+## Instructions
+
+<section id='instructions'>
+Write a function that takes a string as a parameter and returns its the SHA-256 digest.
```

Comment:
Write a function that takes a string as a parameter and returns its SHA-256 digest.